### PR TITLE
[NEAT-305] 🏟 Handle Conversion of Class with 100+ properties

### PR DESCRIPTION
**Pull request by doctrino**

### CODE REVIEW

File: cognite/neat/rules/models/information/_converter.py [hunk 1]
```diff
@@ -93,6 +94,7 @@ def as_dms_architect_rules(self) -> "DMSRules":
             if rule_set:
                 existing_containers.update({c.container for c in rule_set.containers or []})
 
+        containers: list[DMSContainer] = []
```
Comment (doctrino) [inline cognite/neat/rules/models/information/_converter.py:97]: Just moved this one closer to use. This is unrelated to the rest of the PR

**Review by nikokaoja — APPROVED** [review]
